core/dive.c

```diff
@@ -1852,10 +1852,9 @@ static pressure_t merge_pressures(pressure_t a, pressure_t sample_a, pressure_t
 		a = b;
 	if (!b.mbar)
 		b = a;
-	if (take_min && a.mbar < b.mbar)
-		return a;
-	else
-		return b;
+	if(take_min)
```

Comment:
The Subsurface [coding style](https://github.com/subsurface/subsurface/blob/master/CODINGSTYLE.md#basic-rules) is ` if (<condition>)` with a space before the brace.

Reply:
Thank you; fixed.